services/order.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order
+from services.user import get_user_id_by_username

```

Comment:
this creates a dependency of this module on other service. Lets keep them isolated

Reply:
I'll just remove that function since it's short + used only in one function after refactoring